theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/Connector.tsx

```diff
@@ -70,9 +76,21 @@ const Container = styled.div<{isSelected: boolean}>`
 type IProps = Parameters<typeof KeyframeEditor>[0]
 
 const Connector: React.FC<IProps> = (props) => {
-  const {index, trackData} = props
+  const {index, trackData, leaf, selection} = props
+  const copiedKeyframes = getCopiedKeyframes()
```

Comment:
Same comment as above. We can delay these calculations until the context menu is triggered.